spec/services/description_import_spec.rb

```diff
@@ -256,4 +256,433 @@
       expect(updated.value!).to eq expected
     end
   end
+
+  context "when top level contributor" do
+    let(:csv) do
+      CSV.parse(csv_data, headers: true)
+    end
+    let(:expected_hash) do
+      {
+        title: [{value: "my great contributor"}],
+        purl: "https://purl/jr825qh8124"
+      }.tap do |h|
+        h[:contributor] = [expected_contributor] if expected_contributor
+      end
+    end
+    let(:expected) { Cocina::Models::Description.new(expected_hash) }
+
+    [nil, "Someone"].each do |name|
+      context "when role as code, name: '#{name}'" do
```

Comment:
I'm having trouble English-ing this. Can you help translate?

Reply:
a role can be represented as a code or a value (or maybe a URI).    Doesn't everyone know that?

I will try to clarify.